operator/controllers/execution/scans/scan_reconciler.go

```diff
@@ -95,26 +95,33 @@ func (r *ScanReconciler) startScan(scan *executionv1.Scan) error {
 	scan.Status.RawResultType = scanType.Spec.ExtractResults.Type
 	scan.Status.RawResultFile = filepath.Base(scanType.Spec.ExtractResults.Location)
 
+	urlExpirationDuration, err := util.GetUrlExpirationDuration(util.ScanController)
+	if err != nil {
+		r.Log.Error(err, "Failed to parse scan url expiration")
+		panic(err)
+	}
+
+	// this time is hardcoded as its not used internally by the scb so it should be longer lasting
 	findingsDownloadURL, err := r.PresignedGetURL(scan.UID, "findings.json", 7*24*time.Hour)
 	if err != nil {
 		r.Log.Error(err, "Could not get presigned url from s3 or compatible storage provider")
 		return err
 	}
 	scan.Status.FindingDownloadLink = findingsDownloadURL
-	rawResultDownloadURL, err := r.PresignedGetURL(scan.UID, scan.Status.RawResultFile, 7*24*time.Hour)
+	rawResultDownloadURL, err := r.PresignedGetURL(scan.UID, scan.Status.RawResultFile, urlExpirationDuration)
 	if err != nil {
 		return err
 	}
 	scan.Status.RawResultDownloadLink = rawResultDownloadURL
 
-	findingsHeadURL, err := r.PresignedHeadURL(scan.UID, "findings.json", 7*24*time.Hour)
+	findingsHeadURL, err := r.PresignedHeadURL(scan.UID, "findings.json", urlExpirationDuration)
```

Comment:
This one week should be for all urls attached to the scan status not just for the finding json file.
Sorry about that the previous comment wasn't clear on that.